Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM alpine:edge AS builder
+
+RUN apk update \
+    && apk --no-cache add build-base \
+    boost-dev \
+    cmake \
+    pkgconfig \
+    libressl-dev \
+    protobuf-dev \
+    opus-dev \
+    speexdsp-dev \
+    pjproject-dev \
+    # musl doesn't ship execinfo.h
+    libexecinfo-dev \
+    git \
+    && apk --no-cache add log4cpp-dev --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing/
+
+COPY . mumsi/
+
+WORKDIR mumsi
+
+# Build mumlib from git submodule
+RUN mkdir -p modules/mumlib/build && cd modules/mumlib/build \
```

Comment:
building mumlib is already present on the main mumsi cmake list, this isn't necessary.

Reply:
https://github.com/hiro2233/mumsi/blob/master/CMakeLists.txt#L6